src/Persistence/Initialization/Updates/FixEventItemsDropFromMonstersUpdatePlugInSeason6.cs

```diff
@@ -0,0 +1,65 @@
+// <copyright file="FixEventItemsDropFromMonstersUpdatePlugInSeason6.cs" company="MUnique">
+// Licensed under the MIT License. See LICENSE file in the project root for full license information.
+// </copyright>
+
+namespace MUnique.OpenMU.Persistence.Initialization.Updates;
+
+using System.Runtime.InteropServices;
+using MUnique.OpenMU.DataModel.Configuration;
+using MUnique.OpenMU.PlugIns;
+
+/// <summary>
+/// This update fixes event items that have DropsFromMonsters set to true,
+/// but should use dedicated DropItemGroups instead.
+/// </summary>
+[PlugIn(PlugInName, PlugInDescription)]
+[Guid("A8B5C2D1-3E4F-5A6B-7C8D-9E0F1A2B3C4D")]
+public class FixEventItemsDropFromMonstersUpdatePlugInSeason6 : UpdatePlugInBase
+{
+    /// <summary>
+    /// The plug in name.
+    /// </summary>
+    internal const string PlugInName = "Fix Event Items DropsFromMonsters Season 6";
+
+    /// <summary>
+    /// The plug in description.
+    /// </summary>
+    internal const string PlugInDescription = "This update fixes event items that have DropsFromMonsters set to true, causing them to drop at level 0 instead of using their dedicated DropItemGroups.";
+
+    /// <inheritdoc />
+    public override UpdateVersion Version => UpdateVersion.FixEventItemsDropFromMonstersSeason6;
+
+    /// <inheritdoc />
+    public override string DataInitializationKey => VersionSeasonSix.DataInitialization.Id;
+
+    /// <inheritdoc />
+    public override string Name => PlugInName;
+
+    /// <inheritdoc />
+    public override string Description => PlugInDescription;
+
+    /// <inheritdoc />
+    public override bool IsMandatory => true;
+
+    /// <inheritdoc />
+    public override DateTime CreatedAt => new(2025, 01, 27, 10, 0, 0, DateTimeKind.Utc);
+
+    /// <inheritdoc />
+#pragma warning disable CS1998
+    protected override async ValueTask ApplyAsync(IContext context, GameConfiguration gameConfiguration)
+#pragma warning restore CS1998
+    {
+        // Find event items that have DropsFromMonsters = true but also have dedicated DropItemGroups
+        var eventItemsWithDropGroups = gameConfiguration.DropItemGroups
+            .Where(dig => dig.PossibleItems.Any())
+            .SelectMany(dig => dig.PossibleItems)
+            .Where(item => item.Group is 13 or 14 && item.DropsFromMonsters)
+            .Distinct()
+            .ToList();
+
+        foreach (var item in eventItemsWithDropGroups)
+        {
+            item.DropsFromMonsters = false;
+        }
```

Comment:
I debugged this code, and in this list I found another item which needs to be adapted: Symbol of Kundun.
See https://github.com/MUnique/OpenMU/blob/master/src/Persistence/Initialization/VersionSeasonSix/Items/Misc.cs#L56

Reply:
Yes, it's catched by the update plugin, as you mentioned. However, can you adapt the `Misc.cs` for the Symbol of Kundun, too? `item.DropsFromMonsters` needs to be set to `false` here as well.